<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PfdQmlGadgetOptionsPage</class>
 <widget class="QWidget" name="PfdQmlGadgetOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>436</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>457</width>
        <height>436</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>QML file: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Utils::PathChooser" name="qmlSourceFile" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="useOpenGL">
         <property name="text">
          <string>Use OpenGL</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="showTerrain">
         <property name="title">
          <string>Show Terrain:</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="4">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>OsgEarth file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Utils::PathChooser" name="earthFile" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QRadioButton" name="useActualLocation">
            <property name="text">
             <string>Use actual location</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QRadioButton" name="usePredefinedLocation">
            <property name="text">
             <string>Use pre-defined location:</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1" colspan="3">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Latitude:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="latitude"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Longitude:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="longitude"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Altitude:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="altitude"/>
            </item>
           </layout>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="useOnlyCache">
            <property name="text">
             <string>Use only cache data</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>74</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="3">
           <widget class="QPushButton" name="preSeedTerrain">
            <property name="text">
             <string>Pre seed terrain cache</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>121</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header>utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>usePredefinedLocation</sender>
   <signal>toggled(bool)</signal>
   <receiver>latitude</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usePredefinedLocation</sender>
   <signal>toggled(bool)</signal>
   <receiver>longitude</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usePredefinedLocation</sender>
   <signal>toggled(bool)</signal>
   <receiver>altitude</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useOpenGL</sender>
   <signal>toggled(bool)</signal>
   <receiver>showTerrain</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
